predicthq/endpoints/v1/events/schemas.py

```diff
@@ -58,6 +58,30 @@ class Options:
     type = StringType()
     formatted_address = StringType()
 
+class Geometry(Model):
+
+    class Options:
+        serialize_when_none = False
+
+    type = StringType()
+    coordinates = ListType(FloatType())
+
+
+class Geo(Model):
+
+    class Options:
+        serialize_when_none = True
```

Comment:
I don't exactly remember when we need this to be set to True, do you?

Reply:
it's true by default for other general types (string list and etc), so keeping it consistent here

Reply:
this case shouldn't happen tho, it's when `geo` exists but `geometry` doesn't, so could remove the setting potentially, and then `serialize_when_none` would be set to true be default if not specified i believe

Reply:
yes, that makes sense, I'd remove it then.

Reply:
updated